R/batch.R

```diff
@@ -25,7 +25,9 @@ batch <- function(
     batchman.verbose = isTRUE(interactive()),
     stop = FALSE,
     retry = 0,
-    sleep = 0
+    sleep = 0,
+    ncores = parallel::detectCores(),
+    parallel = TRUE
```

Comment:
True that. I'll fix the tests, then flip, then fix it back. apparently need to sacrifice smth for the paralellism